From 31803c0a8ff045b9904dd566aeb9ed4cfa3f5615 Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Thu, 19 Feb 2015 15:40:39 -0800
Subject: [PATCH 3/5] tests: Only run t0501 on 512b disks for now (#1189328)

This fails on 2048 sector disks. Limit it for now.

Related: rhbz#1189328
---
 tests/t0501-duplicate.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tests/t0501-duplicate.sh b/tests/t0501-duplicate.sh
index 9363087..4b59e87 100644
--- a/tests/t0501-duplicate.sh
+++ b/tests/t0501-duplicate.sh
@@ -26,6 +26,9 @@ fi
 PATH="..:$PATH"
 export PATH
 
+# duplicate isn't working right on 2048b sectors
+require_512_byte_sector_size_
+
 for t in msdos gpt bsd; do
     duplicate $t || fail=1
 done
-- 
2.1.0

